ceph/rados/utils.py

```diff
@@ -195,17 +195,23 @@ def set_osd_in(
     return ret_val
 
 
-def osd_remove(ceph_cluster, osd_id, zap=False):
+def osd_remove(ceph_cluster, osd_id, zap=False, force=False):
     """
     osd remove
     Args:
         ceph_cluster: ceph cluster
         osd_id: osd id
         zap: flag to control zapping of device
+        force: flag to remove the OSD forcefully
     """
     config = {"command": "rm", "service": "osd", "pos_args": [osd_id]}
+    cmd_args = {}
     if zap:
-        config["base_cmd_args"] = {"zap": True}
+        cmd_args["zap"] = True
```

Comment:
let's give force as true when zap is true as well.

Reply:
modified the code.